crates/tx-parser/src/utils.rs

```diff
@@ -80,3 +84,12 @@ pub fn generate_taproot_address(
 
     Ok(addr)
 }
+
+/// Reads the operator wallet public keys from Rollup params. Returns None if
+/// not yet bootstrapped
+/// FIXME: This is only for devnet as these pks have to be read from the chain state
```

Comment:
This should not be a docstring:
```suggestion
// FIXME: This is only for devnet as these PKs have to be read from the chain state
```